gh-pages/_layouts/default.html

```diff
@@ -12,6 +12,16 @@
       </div>
     </div>
 
+    <script>
+      document.addEventListener('DOMContentLoaded', function () {
+        new window.SG.Components.TopBar({
```

Comment:
I'm not a huge fan of this sort of manual invocation. It's just boilerplate and variables like logoUrl and logoHref should just be element attributes. Going to look into a different approach for next iteration.